### DISCUSSION TITLE
OCSP support in EJBCA - 6.15.2.1

### DISCUSSION THREAD

**Discussion by sriramec** [Keyfactor/ejbca-ce#401] [General]
We are using EJBCA-CE - 6.15.2.1, is OCSP support available there ?
a. EJBCA should be able to issue certiifcate with OCSP extension - how to configure this ? 
b. OCSP responder able to process OCSP-Revocation-check messages and respond. - how to configure this ?

is the support available in this version or do we need to upgrade ?

**Comment by primetomas** [comment]
Yes there is OCSP in EJBCA 6.15. 
But no you should absolutely not run that old version. It's five years old and hence have known security issues. You should always keep all your software updated, not only EJBCA.
